core/services/fluxmonitor/flux_monitor.go

```diff
@@ -548,6 +615,45 @@ func (p *PollingDeviationChecker) consume() {
 	}
 }
 
+func (p *PollingDeviationChecker) performInitialPoll() {
+	if !p.initr.PollTimer.Disabled && !p.isHibernating {
+		p.pollIfEligible(DeviationThresholds{
+			Rel: float64(p.initr.Threshold),
+			Abs: float64(p.initr.AbsoluteThreshold),
+		})
+	}
+}
+
+func (p *PollingDeviationChecker) setPollTicker() {
+	if p.pollTicker != nil {
+		p.pollTicker.Stop()
+	}
+	if !p.initr.PollTimer.Disabled && !p.isHibernating {
+		p.pollTicker = time.NewTicker(p.initr.PollTimer.Period.Duration())
+		p.chPollTicker = p.pollTicker.C
+	}
+}
+
+// Hibernate restarts the PollingDeviationChecker in hibernation mode
+func (p *PollingDeviationChecker) Hibernate() {
```

Comment:
It looks like this and Reactivate are only called from inside FM.  Can we make this private (and maybe export them to our tests in helpers_test.go)?